src/mmx.cc

```diff
@@ -13,7 +13,7 @@ bool mmxIsSupported()
 
     // TODO: There are other ways to determine MMX using FLAGS register.
 
-#ifdef _WIN32
+#if defined (_MSC_VER)
```

Comment:
I haven't tested it but there is a [standard inline assembly declaration](https://en.cppreference.com/w/cpp/language/asm). Perhaps, this could work for both MSVC and Mingw on Windows:

```c++
#if defined (_WIN32)
    asm(R"(
        mov eax, 1
        cpuid
        and edx, 0x800000
        mov v1, edx
    )");
#else
    v1 = 0;
#endif
```

Reply:
Tried it on MinGW, won't work.

...\AppData\Local\Temp\cc0OAfeX.s: Assembler messages:
...\AppData\Local\Temp\cc0OAfeX.s:22: Error: too many memory references
for `mov'
...\AppData\Local\Temp\cc0OAfeX.s:24: Error: too many memory references
for `and'
...\AppData\Local\Temp\cc0OAfeX.s:25: Error: too many memory references
for `mov'
mingw32-make[2]: *** [CMakeFiles\fallout2-ce.dir\build.make:991:
CMakeFiles/fallout2-ce.dir/src/mmx.cc.obj] Error 1